web-local/src/lib/components/assets/TableAssets.svelte

```diff
@@ -106,14 +107,11 @@
     }
   };
 
-  const deleteSource = (tableName: string) => {
-    const currentAssetIndex = $persistentTableStore.entities.findIndex(
-      (entity) => entity.tableName === tableName
-    );
+  const deleteSource = (tableName: string, id: string) => {
```

Comment:
We can remove this `tableName` parameter now

Reply:
It used to delete the table in `deleteSourceApi`